python/graphstorm/dataloading/dataloading.py

```diff
@@ -182,6 +185,20 @@ class GSgnnEdgeDataLoader(GSgnnEdgeDataLoaderBase):
         The node types that requires to construct node features.
     construct_feat_fanout : int
         The fanout required to construct node features.
+        
+    Examples
+    ------------
+    To train a 2-layer GNN for edge prediction on a set of edges ``target_idx`` on 
+    a homogenous graph where each nodes takes messages from 15 neighbors on the first layer
```

Comment:
we don't need to say "homogeneous" because the `target_idx` could be a dictionary for heterogenous graphs.